1.9/administration/debugging/index.md

```diff
@@ -4,19 +4,30 @@ feature_maturity: experimental
 menu_order: 3.3
 ---
 
-When things go wrong, you need to see the logs to understand how to fix it. DC/OS services and tasks write `stdout` and `stderr` files in their sandboxes by default. Traditionally, log aggregation has been the solution here. Write the logs locally and then ship all that data elsewhere for someone to actually access. Moving data around is expensive, especially when it ends up being written multiple times.
+The `dcos task exec` DC/OS CLI command allows you to execute commands inside of a task's container. All you will need is to be logged into the DC/OS CLI via `dcos auth login`, the ID of the task, and a running container. The `dcos task exec` command offers an experience very similar to [`docker exec`](https://docs.docker.com/engine/reference/commandline/exec/), without any need for SSH keys. The connection occurs over an HTTP stream and does not use SSH at all.
```

Comment:
From what I understand, the fact that you don't need SSH keys is a huge feature. I can see what you mean about going into the protocol. Instinctively, I would agree. At the same time, I have observed people showing great interest in this. Let's drop it but I think we should keep at least the SSH key mention. Thanks @klueska! 👍